app/controllers/nonsipp/employercontributions/EmployerContributionsMemberListController.scala

```diff
@@ -227,13 +226,18 @@ object EmployerContributionsMemberListController {
     FormPageViewModel(
       title = Message(title, memberList.size),
       heading = Message(heading, memberList.size),
-      description = Some(ParagraphMessage("employerContributions.MemberList.paragraph")),
+      description = None,
       page = ActionTableViewModel(
-        inset = "employerContributions.MemberList.inset",
+        inset = ParagraphMessage(
```

Comment:
Are we sure that we are going to show "You must tell us about ......" always ??
I'm asking this because some of the journey we have fixed in this sprint, they are conditional. If you reach the maximum, we were hiding this and showing an warning message instead.

FYI @harishmuddegowda

Reply:
Yes, I'm pretty sure. For member payments section this should be always visible.
For other journeys there is some optional inset text, which is shown when some kind of maximum is reached.